setup.py

```diff
@@ -95,7 +95,7 @@
 # 1. all dependencies should be listed here with their version requirements if any
 # 2. once modified, run: `make deps_table_update` to update src/transformers/dependency_versions_table.py
 _deps = [
-    "Pillow<10.0.0",
+    "Pillow>=10.0.1,<=11.0",
```

Comment:
Why `<=11.0`?

Reply:
I'll pin <=15.0, it's just to not allow whatever! 
Thanks for the tips will run the slow model tests 😉

Reply:
- [x] Vit
- [x] Clip
- [x] Detr (tests/models/detr/test_modeling_detr.py::DetrModelIntegrationTestsTimmBackbone::test_inference_panoptic_segmentation_head  fails:
```python 
E       - {'id': 1, 'label_id': 17, 'score': 0.994097, 'was_fused': False}
E       ?                                                                ^
E       
E       + {'id': 1, 'label_id': 17, 'score': 0.994096, 'was_fused': False}
```
Might be my arch rather than anything
- [x] SAM

Which models would you think rely a lot on pillow version that I should test as well?

Reply:
I think that difference is definitely in the realm of acceptable! 

All vision models rely on Pillow for their processing. I picked these models to cover different default resampling algos + popular models on different tasks. AFAICT - there's just bicubic, bilinear and nearest used for resizing settings and they've all been tested :)